.github/workflows/molecule.yml

```diff
@@ -14,8 +14,16 @@ jobs:
 
     strategy:
       matrix:
-        distro: [rockylinux8, rockylinux9, ubuntu2204]
-        scenario: [default, renew, ca-renew]
+        distro:
+          - rockylinux8
+          - rockylinux9
```

Comment:
Rocky 8 is already EOL. https://endoflife.date/rocky-linux  
What about adding Rocky 10 instead? There is an rocky 10 from geerlingguy already.

Reply:
Rockylinx 10 was planned for another PR. 😆 Last time I looked I must have mixed up EOL and "end of security support". So I thought, it would still be supported.
I'll push another commit changing 8 to 10.